### ISSUE TITLE
Move the AWS SIEM to core account space

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#850]
**Describe the issue**
The AWS SIEM demo is in workload account space and doesn't have access it needs to work properly.  We wish to install it in the core account space so it has full access.

AWS provides an open source log analytics dashboard capable of ingesting and visualizing a variety of AWS logs in a user friendly online dashboard.
We wish to utilize this solution as a central dashboard for all security events across the SEA Landing Zone.

**Which Sprint Goal is this issue related to?**
> Note: 'Milestone' is a ZenHub term that we use synonymously with 'Sprint'. 

The 'Milestone' description (created in ZenHub) should clearly list Sprint Goals. This section should indicate which 'goal' this issue is related to.

**Additional context**
- Start by doing this as an overlay
- AWS SIEM dashboard is deployed to a designated account (e.g. Reporting account)
- SEA Logs in Log-Archive account are pushed to the SIEM dashboard for data analytics and reporting
- The desired dashboards are populated with data regularly
- Initial users/groups are created and permissions granted 
- AWS SIEM can populate graphs and is resident in core account space.

**Comment by sheaphillips** [comment]
Consulted with NRM OneTeam Tech lead for additional background on the need described in this ticket. The team is taking the approach of building a PoC over the next several months and has made arrangement to avoid/reduce costs so shuffling their resources around to reduce cost is not a priority from their perspective. Consulted CPF PO and concluded that this ticket could be closed and perhaps the concern re-evaluated in ~6 months once the PoC is complete or further along.
